geopandas/tests/test_op_output_types.py

```diff
@@ -238,12 +226,13 @@ def identity(x):
     assert_object(df[["value1", "value2"]].apply(identity), pd.DataFrame)
     assert_object(df[[geo_name, "geometry2"]].apply(identity), GeoDataFrame, geo_name)
     assert_object(df[[geo_name]].apply(identity), GeoDataFrame, geo_name)
-    expected_geo_col_name = None if compat.PANDAS_GE_14 else "geometry"
-    assert_obj_no_active_geo_col(
-        df[["geometry2", "value1"]].apply(identity), GeoDataFrame, expected_geo_col_name
-    )
+    geo_name_none = geo_name if compat.PANDAS_GE_14 else "geometry"
```

Comment:
```suggestion
    expected_geo_col_name = geo_name if compat.PANDAS_GE_14 else "geometry"
```

the "none" is a bit confusing?